src/main/java/br/edu/ufcg/computacao/alumni/core/MatchesFinder.java

```diff
@@ -162,12 +172,12 @@ private int getScoreFromName(String alumniName, String linkedinName) {
 
 		int score = 0;
 		
-		if (alumniParsedName.isComposed()) {
+		if (alumniParsedName.isComposed() && !linkedinParsedName.isComposed()) {
 			linkedinParsedName.turnComposed();
 		}
 		
-		score += compareNames(alumniParsedName.getNames(), linkedinParsedName.getNames());
-		score += compareNames(alumniParsedName.getSurnames(), linkedinParsedName.getSurnames());
+		score += compareNames(alumniParsedName.getNames(), linkedinParsedName.getNames(), 20);
+		score += compareNames(alumniParsedName.getSurnames(), linkedinParsedName.getSurnames(), 10);
```

Comment:
Isso resolve aquele caso da comparação de Hugo Ranieri di Tal e Tal com Hugo Ranieri? O problema era que na parte alumni o name fica hugo,ranieri e o surname tal,tal, enquanto que na parte linkedin fica name hugo, surname ranieri. Não seria o caso de fazer 4 comparações, com bases diferentes:
compareNames(alumniParsedName.getnames(), linkedinParsedName.getNames, 20);
compareNames(alumniParsedName.getSurnames(), linkedinParsedName.getSurnames, 20);
compareNames(alumniParsedName.getnames(), linkedinParsedName.getSurnames, 10);
compareNames(alumniParsedName.getSurnames(), linkedinParsedName.getNames, 10);

Reply:
O que resolve isso é a verificação logo acima, em que se o nome do aluno for composto mas o do linkedin nao for, ele torna o do linkedin composto. Eu optei pelo peso maior no nome pois achei mais pertinente o(s) nome(s) principal(is) valerem mais.

Reply:
Ok.